frontend/catalyst/jax_primitives.py

```diff
@@ -1327,7 +1327,7 @@ def _qunitary_lowering(
 @qmeasure_p.def_abstract_eval
 def _qmeasure_abstract_eval(qubit, postselect: int = None):
     assert isinstance(qubit, AbstractQbit)
-    return core.ShapedArray((), bool), qubit
+    return core.DShapedArray((), np.dtype("bool")), qubit
```

Comment:
This doesn't need to be dynamic given that the tensor is rank 0.

Reply:
Done